migrations/20240529091617-wishlists.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    /**
```

Comment:
there are not table inside this migration

Reply:
let me delete it